src/components/button/Button.js

```diff
@@ -0,0 +1,24 @@
+import './button.scss';
+
+export class Button extends React.Component {
+  constructor() {
+    super();
+    this.state = { btn: '' };
+  }
+
+  changeClass() {
+    if (this.state.btn === 'active') {
+      this.setState({ btn: '' });
+    } else {
+      this.setState({ btn: 'active' });
+    }
+  }
+
+  render() {
+    return (
+      <button className={this.state.btn} onClick={() => this.changeClass()}>
```

Comment:
why don't use just `onClick={this.changeClass}`?

Reply:
@iliatcymbal changeClass was a class method, it did not work, fixed this